src/components/NotificationRow.tsx

```diff
@@ -154,7 +154,7 @@ export const NotificationRow: FC<IProps> = ({ notification }) => {
           <div title={reason.description}>{reason.title}</div>
           <div title={updatedLabel}>{updatedAt}</div>
           {settings.showPills && (
-            <div>
+            <div className="flex">
```

Comment:
@setchy this actually fixes the pill regression! Was supposed to be on the other PR :(

Reply:
Ah, got it. I'll let you raise the fix with this and I'll close mine

Reply:
Though this does mean that the PillButton isn't usable standalone and requires the consuming component to apply that...

Reply:
I understand what you are saying, but the use of gap is always superior to the use of margins.